newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/NrqlCondition.java

```diff
@@ -61,6 +63,15 @@
     @NonNull
     private SinceValue sinceValue;
 
+    /**
+     * NRQL Signal configuration.
+     */
+    private NrqlSignalConfiguration signal;
+
+    public Optional<NrqlSignalConfiguration> getSignal() {
```

Comment:
We don't return Optional in similar cases. I know it's tempting, and I agree that it may be good idea to return Optional for nullable values, but I tresure API consistency even more :)

We are using Optionals only in Policy, and only for collections. Don't know the reason for that, but again - I prefer consistency in this case

Reply:
Sure :)